src/libs/actions/App.ts

```diff
@@ -202,6 +202,7 @@ function getPolicyParamsForOpenOrReconnect(): Promise<PolicyParamsForOpenOrRecon
                     resolve({policyIDList: getNonOptimisticPolicyIDs(policies)});
                 },
             });
+            Log.info('[getPolicyParamsForOpenOrReconnect] isReadyToOpenApp resolved', true, {policyIDListConnectionID: connectionID});
```

Comment:
Not really sure why I'm logging the connectionID, I guess in case we find that the policyIDList promise isn't resolving we can see what's going on with that connection? Don't really know how to check

Reply:
I don't think it will help. The connectionIDs would only help you see the connection in the Onyx mapping - which doesn't really do much except hold the callback.

Reply:
Maybe put this log above the `Onyx.connect()` call.